assignment/assignment-api/api/src/java/org/sakaiproject/assignment/api/AssignmentService.java

```diff
@@ -912,4 +912,10 @@ public void removeAssignmentContent(AssignmentContentEdit content) throws Assign
 	 * Params: AssignmentSubmission s
 	 */
 	public boolean assignmentUsesAnonymousGrading(Assignment a);
+	
+	/**
+	 * Get Scale Factor from the property xxxxx
```

Comment:
Replace xxxxx with the actual sakai.property this method checks, assignment.grading.decimals